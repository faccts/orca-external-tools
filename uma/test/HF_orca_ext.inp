! extopt opt
%method
  progext "../../uma.sh"
end
*xyz 0 1
  H 0 0 0
  F 0 0 0.9
*
