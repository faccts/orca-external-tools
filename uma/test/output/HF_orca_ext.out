
                                 *****************
                                 * O   R   C   A *
                                 *****************

                                            #,                                       
                                            ###                                      
                                            ####                                     
                                            #####                                    
                                            ######                                   
                                           ########,                                 
                                     ,,################,,,,,                         
                               ,,#################################,,                 
                          ,,##########################################,,             
                       ,#########################################, ''#####,          
                    ,#############################################,,   '####,        
                  ,##################################################,,,,####,       
                ,###########''''           ''''###############################       
              ,#####''   ,,,,##########,,,,          '''####'''          '####       
            ,##' ,,,,###########################,,,                        '##       
           ' ,,###''''                  '''############,,,                           
         ,,##''                                '''############,,,,        ,,,,,,###''
      ,#''                                            '''#######################'''  
     '                                                          ''''####''''         
             ,#######,   #######,   ,#######,      ##                                
            ,#'     '#,  ##    ##  ,#'     '#,    #''#        ,####,   ,####,        
            ##       ##  ##   ,#'  ##            #'  '#       #'       #'  '#        
            ##       ##  #######   ##           ,######,      #####,   #    #        
            '#,     ,#'  ##    ##  '#,     ,#' ,#      #,     #,   #   #,  ,#        
             '#######'   ##     ##  '#######'  #'      '#     '####' # '####'        



                #########################################################
                #                        -***-                          #
                #          Department of theory and spectroscopy        #
                #                                                       #
                #                      Frank Neese                      #
                #                                                       #
                #     Directorship, Architecture, Infrastructure        #
                #                    SHARK, DRIVERS                     #
                #        Core code/Algorithms in most modules           #
                #                                                       #
                #        Max Planck Institute fuer Kohlenforschung      #
                #                Kaiser Wilhelm Platz 1                 #
                #                 D-45470 Muelheim/Ruhr                 #
                #                      Germany                          #
                #                                                       #
                #                  All rights reserved                  #
                #                        -***-                          #
                #########################################################


                         Program Version 6.0.1  -  RELEASE   -


 With contributions from (in alphabetic order):
[Max-Planck-Institut fuer Kohlenforschung]                               
  Daniel Aravena         : Magnetic Suceptibility                        
  Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
  Alexander A. Auer      : GIAO ZORA, VPT2 properties, NMR spectrum      
  Ute Becker             : All parallelization in ORCA, NUMFREQ, NUMCALC 
  Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLD           
  Martin Brehm           : Molecular dynamics                            
  Dmytro Bykov           : pre 5.0 version of the SCF Hessian            
  Marcos Casanova-PÃ¡ez   : Triplet and SCS-CIS(D). UHF-(DLPNO)-IP/EA/STEOM-CCSD. UHF-CVS-IP/STEOM-CCSD   
  Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE 
  Pauline Colinet        : FMM embedding                                 
  Dipayan Datta          : RHF DLPNO-CCSD density                        
  Achintya Kumar Dutta   : EOM-CC, STEOM-CC                              
  Nicolas Foglia         : Exact transition moments, OPA infrastructure, MCD improvements    
  Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI      
  Miquel Garcia-Rates    : C-PCM and meta-GGA Hessian, CCSD/C-PCM, Gaussian charge scheme 
  Tiago L. C. Gouveia    : GS-ROHF, GS-ROCIS                             
  Yang Guo               : DLPNO-NEVPT2, F12-NEVPT2, CIM, IAO-localization        
  Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods 
  Ingolf Harden          : AUTO-CI MPn and infrastructure                
  Benjamin Helmich-Paris : MC-RPA, TRAH-(SCF,CASSCF), AVAS, COSX integrals, SCF dyn. polar. 
  Lee Huntington         : MR-EOM, pCC                                   
  Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM     
  Riya Kayal             : Wick's Theorem for AUTO-CI, AUTO-CI UHF-CCSDT 
  Emily Kempfer          : AUTO-CI, RHF CISDT and CCSDT                  
  Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density, CASPT2, CASPT2-K, improved NEVPT2 
  Axel Koslowski         : Symmetry handling                             
  Simone Kossmann        : meta-GGA functionals, TD-DFT gradient, OOMP2, (MP2 Hessian; deprecated post 5.0) 
  Lucas Lang             : DCDCAS                                        
  Marvin Lechner         : AUTO-CI (C++ implementation), FIC-MRCC        
  Spencer Leger          : CASSCF response                               
  Dagmar Lenk            : GEPOL surface, SMD, ORCA-2-JSON               
  Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization 
  Dimitrios Manganas     : Further ROCIS development; embedding schemes. LFT, Crystal Embedding 
  Dimitrios Pantazis     : SARC Basis sets                               
  Anastasios Papadopoulos: AUTO-CI, single reference methods and gradients 
  Taras Petrenko         : pre 6.0 DFT Hessian and TD-DFT gradient, (ASA, deprecated), ECA, 1-Electron XAS/XES, NRVS 
  Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient                 
  Christoph Reimann      : Effective Core Potentials                     
  Marius Retegan         : Local ZFS, SOC                                
  Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples 
  Michael Roemelt        : Original ROCIS implementation                 
  Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density      
  Barbara Sandhoefer     : DKH picture change effects                    
  Kantharuban Sivalingam : CASSCF convergence/infrastructure, NEVPT2 and variants, FIC-MRCI 
  Bernardo de Souza      : ESD, SOC TD-DFT                               
  Georgi L. Stoychev     : AutoAux, RI-MP2 NMR, DLPNO-MP2 response, X2C  
  Van Anh Tran           : RI-MP2 g-tensors                              
  Willem Van den Heuvel  : Paramagnetic NMR                              
  Zikuan Wang            : NOTCH, Electric field optimization            
  Frank Wennmohs         : Technical directorship and infrastructure     
  Hang Xu                : AUTO-CI-Response properties                   
                                                                         
[FACCTs GmbH]                                                            
  Markus Bursch, Miquel Garcia-Rates, Christoph Riplinger, Bernardo de Souza, Georgi L. Stoychev 
                                                                 
  APM, Basis sets (HGBS, AHGBS, def-TZVP (Ac-Lr), def2-XVPD (La-Lu)), CI-OPT, improved COSX, DLPNO-Multilevel,           
  DOCKER, DRACO, updates on ESD, GOAT, IRC, LR-CPCM, MBIS, meta-GGA TD-DFT gradient, ML-optimized integration grids, 
  MM, NACMEs, nearIR, NEB, NEB-TS, NL-DFT gradient (VV10), 2- and 3-layer-ONIOM, interface openCOSMO-RS, QMMM,   
  Crystal-QMMM, SF, symmetry and pop. for TD-DFT, r2SCAN hybrids, SOLVATOR                                   
                                                                         
[Other institutions]                                                     
  V. Asgeirsson          : NEB                                           
  Christoph Bannwarth    : sTDA-DFT, sTD-DFT, PBEh-3c, B97-3c, D3        
  Sebastian Ehlert       : rSCAN, r2SCAN, r2SCAN-3c, D4, dhf basis sets  
  Marvin Friede          : D4 for Fr, Ra, Ac-Lr                          
  Lars Goerigk           : TD-DFT with DH, B97 family of functionals     
  Stefan Grimme          : VdW corrections, initial TS optimization, DFT functionals, gCP, sTDA/sTD-DF 
  Waldemar Hujo          : DFT-NL                                        
  H. Jonsson             : NEB                                           
  Holger Kruse           : gCP                                           
  Marcel Mueller         : wB97X-3c, vDZP basis set                      
  Hagen Neugebauer       : wr2SCAN                                       
  Tobias Risthaus        : range-separated hybrid DFT                    
  Lukas Wittmann         : regularized MP2, r2SCAN double-hybrids, wr2SCAN        
                                                                         
We gratefully acknowledge several colleagues who have allowed us to      
interface, adapt or use parts of their codes:                            
  Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods 
  Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG                   
  Ulf Ekstrom                                   : XCFun DFT Library      
  Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
  Frank Weinhold                                : gennbo (NPA and NBO analysis)           
  Simon Mueller                                 : openCOSMO-RS                            
  Christopher J. Cramer and Donald G. Truhlar   : smd solvation model                     
  S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library                           
  Liviu Ungur et al                             : ANISO software                          


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 6.2.2
 For citations please refer to: https://libxc.gitlab.io

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines
   Shared memory     :  Shared parallel matrices
   BLAS/LAPACK       :  OpenBLAS 0.3.27  USE64BITINT DYNAMIC_ARCH NO_AFFINITY Haswell SINGLE_THREADED
        Core in use  :  Haswell
   Copyright (c) 2011-2014, The OpenBLAS Project


================================================================================

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


WARNING: Geometry Optimization
  ===> : Switching off AutoStart
         For restart on a previous wavefunction, please use MOREAD

Using external energy/gradient program given in input:
  "../../uma.sh"


WARNING: four-center integrals in TRAH must be done with SHARK!
  ===> : Turning Shark on


================================================================================
                                       INPUT FILE
================================================================================
NAME = HF_orca_ext.inp
|  1> ! extopt opt
|  2> %method
|  3>   progext "../../uma.sh"
|  4> end
|  5> *xyz 0 1
|  6>   H 0 0 0
|  7>   F 0 0 0.9
|  8> *
|  9> 
| 10>                          ****END OF INPUT****
================================================================================

                       *****************************
                       * Geometry Optimization Run *
                       *****************************

Geometry optimization settings:
Update method            Update   .... BFGS
Choice of coordinates    CoordSys .... (2022) Redundant Internals
Initial Hessian          InHess   .... Almloef's Model
Max. no of cycles        MaxIter  .... 50

Convergence Tolerances:
Energy Change            TolE     ....  5.0000e-06 Eh
Max. Gradient            TolMAXG  ....  3.0000e-04 Eh/bohr
RMS Gradient             TolRMSG  ....  1.0000e-04 Eh/bohr
Max. Displacement        TolMAXD  ....  4.0000e-03 bohr
RMS Displacement         TolRMSD  ....  2.0000e-03 bohr
Strict Convergence                .... False

------------------------------------------------------------------------------
                        ORCA OPTIMIZATION COORDINATE SETUP
------------------------------------------------------------------------------

The optimization will be done in redundant internal coordinates (2022)
Making redundant internal coordinates   ...  (2022 redundants) done
Evaluating the initial hessian          ...  (Almloef) done
Evaluating the coordinates              ...  done
Calculating the B-matrix                .... done
Calculating the G-matrix                .... done
Diagonalizing the G-matrix              .... done
The first mode is                       ....    0
The number of degrees of freedom        ....    1

    -----------------------------------------------------------------
                    Redundant Internal Coordinates


    -----------------------------------------------------------------
         Definition                    Initial Value    Approx d2E/dq
    -----------------------------------------------------------------
      1. B(F   1,H   0)                  0.9000         0.602260   
    -----------------------------------------------------------------

Number of atoms                         .... 2
Number of degrees of freedom            .... 1


         *************************************************************
         *                GEOMETRY OPTIMIZATION CYCLE   1            *
         *************************************************************
---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000    0.000000
  F      0.000000    0.000000    0.900000

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 H     1.0000    0     1.008    0.000000    0.000000    0.000000
   1 F     9.0000    0    18.998    0.000000    0.000000    1.700754

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     0.900000000000     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     1.700753520529     0.00000000     0.00000000

Calling ../../uma.sh HF_orca_ext_EXT.extinp.tmp   ... 

Init time:    2.379 seconds
Calc time:    0.000 seconds
Import time:  1.786 seconds
Total time:   4.252 seconds

External program completed successfully in 4.897 sec

-------------------------------------------------   --------------------
FINAL SINGLE POINT ENERGY (From external program)      -100.461425722700
-------------------------------------------------   --------------------


------------------
CARTESIAN GRADIENT
------------------

   1   H   :    0.000000000   -0.000000000    0.029673452
   2   F   :   -0.000000000    0.000000000   -0.029673452

Difference to translation invariance:
           :    0.0000000000    0.0000000000    0.0000000000

Difference to rotation invariance:
           :   -0.0000000000   -0.0000000000    0.0000000000

Norm of the Cartesian gradient     ...    0.0419645976
RMS gradient                       ...    0.0171319752
MAX gradient                       ...    0.0296734516
------------------------------------------------------------------------------
                         ORCA GEOMETRY RELAXATION STEP
------------------------------------------------------------------------------

Reading the OPT-File                    .... done
Getting information on internals        .... done
Copying old internal coords+grads       .... done
Making the new internal coordinates     .... (2022 redundants) done
Validating the new internal coordinates .... (2022 redundants) done
Calculating the B-matrix                .... done
Calculating the G,G- and P matrices     .... done
Transforming gradient to internals      .... done
Projecting the internal gradient        .... done
Number of atoms                         ....   2
Number of internal coordinates          ....   1
Current Energy                          ....  -100.461425723 Eh
Current gradient norm                   ....     0.041964598 Eh/bohr
Maximum allowed component of the step   ....  0.300
Current trust radius                    ....  0.300
Evaluating the initial hessian          ....  (Almloef) done
Projecting the Hessian                  .... done
Forming the augmented Hessian           .... done
Diagonalizing the augmented Hessian     .... done
Last element of RFO vector              ....  0.998794268
Lowest eigenvalues of augmented Hessian:
 -0.001458483
Length of the computed step             ....  0.049151121
The final length of the internal step   ....  0.049151121
Converting the step to Cartesian space:
 Initial RMS(Int)=    0.0491511207
Transforming coordinates:
 Iter   0:  RMS(Cart)=    0.0141887064 RMS(Int)=    0.0491511207
done
Storing new coordinates                 .... done

                                .--------------------.
          ----------------------|Geometry convergence|-------------------------
          Item                value                   Tolerance       Converged
          ---------------------------------------------------------------------
          RMS gradient        0.0296734516            0.0001000000      NO
          MAX gradient        0.0296734516            0.0003000000      NO
          RMS step            0.0491511207            0.0020000000      NO
          MAX step            0.0491511207            0.0040000000      NO
          ........................................................
          Max(Bonds)      0.0260      Max(Angles)    0.00
          Max(Dihed)        0.00      Max(Improp)    0.00
          ---------------------------------------------------------------------

The optimization has not yet converged - more geometry cycles are needed


    ---------------------------------------------------------------------------
                         Redundant Internal Coordinates
                            (Angstroem and degrees)

        Definition                    Value    dE/dq     Step     New-Value
    ----------------------------------------------------------------------------
     1. B(F   1,H   0)                0.9000 -0.029673  0.0260    0.9260   
    ----------------------------------------------------------------------------

Geometry step timings:
Preparation and reading OPT file:      0.000 s (10.956 %)
Internal coordinates            :      0.000 s ( 1.632 %)
B/P matrices and projection     :      0.000 s (14.452 %)
Hessian update/contruction      :      0.000 s (11.189 %)
Making the step                 :      0.000 s (40.326 %)
Converting the step to Cartesian:      0.000 s ( 4.196 %)
Storing new data                :      0.000 s ( 2.564 %)
Checking convergence            :      0.000 s ( 0.000 %)
Final printing                  :      0.000 s (13.287 %)
Total time                      :      0.000 s

Time for energy+gradient        :      4.900 s
Time for complete geometry iter :      4.918 s

         *************************************************************
         *                GEOMETRY OPTIMIZATION CYCLE   2            *
         *************************************************************
---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000   -0.013005
  F      0.000000    0.000000    0.913005

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 H     1.0000    0     1.008    0.000000    0.000000   -0.024576
   1 F     9.0000    0    18.998    0.000000    0.000000    1.725329

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     0.926009652821     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     1.749904641198     0.00000000     0.00000000

Calling ../../uma.sh HF_orca_ext_EXT.extinp.tmp   > HF_orca_ext.lastext ...  done

External program completed successfully in 4.849 sec
------------------------------------------------------------------------------
                         ORCA GEOMETRY RELAXATION STEP
------------------------------------------------------------------------------

Reading the OPT-File                    .... done
Getting information on internals        .... done
Copying old internal coords+grads       .... done
Making the new internal coordinates     .... (2022 redundants) done
Validating the new internal coordinates .... (2022 redundants) done
Calculating the B-matrix                .... done
Calculating the G,G- and P matrices     .... done
Transforming gradient to internals      .... done
Projecting the internal gradient        .... done
Number of atoms                         ....   2
Number of internal coordinates          ....   1
Current Energy                          ....  -100.462063763 Eh
Current gradient norm                   ....     0.003844435 Eh/bohr
Maximum allowed component of the step   ....  0.300
Current trust radius                    ....  0.300
Updating the Hessian (BFGS)             .... done
Forming the augmented Hessian           .... done
Diagonalizing the augmented Hessian     .... done
Last element of RFO vector              ....  0.999991493
Lowest eigenvalues of augmented Hessian:
 -0.000011213
Length of the computed step             ....  0.004124843
The final length of the internal step   ....  0.004124843
Converting the step to Cartesian space:
 Initial RMS(Int)=    0.0041248428
Transforming coordinates:
 Iter   0:  RMS(Cart)=    0.0011907396 RMS(Int)=    0.0041248428
done
Storing new coordinates                 .... done
The predicted energy change is          ....    -0.000005607
Previously predicted energy change      ....    -0.000731003
Actually observed energy change         ....    -0.000638041
Ratio of predicted to observed change   ....     0.872828469
New trust radius                        ....     0.450000000

                                .--------------------.
          ----------------------|Geometry convergence|-------------------------
          Item                value                   Tolerance       Converged
          ---------------------------------------------------------------------
          Energy change      -0.0006380406            0.0000050000      NO
          RMS gradient        0.0027184258            0.0001000000      NO
          MAX gradient        0.0027184258            0.0003000000      NO
          RMS step            0.0041248428            0.0020000000      NO
          MAX step            0.0041248428            0.0040000000      NO
          ........................................................
          Max(Bonds)      0.0022      Max(Angles)    0.00
          Max(Dihed)        0.00      Max(Improp)    0.00
          ---------------------------------------------------------------------

The optimization has not yet converged - more geometry cycles are needed


    ---------------------------------------------------------------------------
                         Redundant Internal Coordinates
                            (Angstroem and degrees)

        Definition                    Value    dE/dq     Step     New-Value
    ----------------------------------------------------------------------------
     1. B(F   1,H   0)                0.9260  0.002718 -0.0022    0.9238   
    ----------------------------------------------------------------------------

Geometry step timings:
Preparation and reading OPT file:      0.000 s ( 1.269 %)
Internal coordinates            :      0.000 s ( 0.282 %)
B/P matrices and projection     :      0.000 s ( 1.975 %)
Hessian update/contruction      :      0.000 s ( 3.338 %)
Making the step                 :      0.000 s ( 0.705 %)
Converting the step to Cartesian:      0.000 s ( 0.376 %)
Storing new data                :      0.000 s ( 0.376 %)
Checking convergence            :      0.000 s ( 0.141 %)
Final printing                  :      0.002 s (91.443 %)
Total time                      :      0.002 s

Time for energy+gradient        :      4.850 s
Time for complete geometry iter :      4.866 s

         *************************************************************
         *                GEOMETRY OPTIMIZATION CYCLE   3            *
         *************************************************************
---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000   -0.011913
  F      0.000000    0.000000    0.911913

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 H     1.0000    0     1.008    0.000000    0.000000   -0.022513
   1 F     9.0000    0    18.998    0.000000    0.000000    1.723267

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     0.923826880022     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     1.745779798397     0.00000000     0.00000000

Calling ../../uma.sh HF_orca_ext_EXT.extinp.tmp   > HF_orca_ext.lastext ...  done

External program completed successfully in 4.873 sec
------------------------------------------------------------------------------
                         ORCA GEOMETRY RELAXATION STEP
------------------------------------------------------------------------------

Reading the OPT-File                    .... done
Getting information on internals        .... done
Copying old internal coords+grads       .... done
Making the new internal coordinates     .... (2022 redundants) done
Validating the new internal coordinates .... (2022 redundants) done
Calculating the B-matrix                .... done
Calculating the G,G- and P matrices     .... done
Transforming gradient to internals      .... done
Projecting the internal gradient        .... done
Number of atoms                         ....   2
Number of internal coordinates          ....   1
Current Energy                          ....  -100.462069845 Eh
Current gradient norm                   ....     0.000317496 Eh/bohr
Maximum allowed component of the step   ....  0.300
Current trust radius                    ....  0.450
Updating the Hessian (BFGS)             .... done
Forming the augmented Hessian           .... done
Diagonalizing the augmented Hessian     .... done
Last element of RFO vector              ....  0.999999931
Lowest eigenvalues of augmented Hessian:
 -0.000000083
Length of the computed step             ....  0.000371319
The final length of the internal step   ....  0.000371319
Converting the step to Cartesian space:
 Initial RMS(Int)=    0.0003713193
Transforming coordinates:
 Iter   0:  RMS(Cart)=    0.0001071906 RMS(Int)=    0.0003713193
done
Storing new coordinates                 .... done
The predicted energy change is          ....    -0.000000042
Previously predicted energy change      ....    -0.000005607
Actually observed energy change         ....    -0.000006082
Ratio of predicted to observed change   ....     1.084732644
New trust radius                        ....     0.675000000

                                .--------------------.
          ----------------------|Geometry convergence|-------------------------
          Item                value                   Tolerance       Converged
          ---------------------------------------------------------------------
          Energy change      -0.0000060817            0.0000050000      NO
          RMS gradient        0.0002245035            0.0001000000      NO
          MAX gradient        0.0002245035            0.0003000000      YES
          RMS step            0.0003713193            0.0020000000      YES
          MAX step            0.0003713193            0.0040000000      YES
          ........................................................
          Max(Bonds)      0.0002      Max(Angles)    0.00
          Max(Dihed)        0.00      Max(Improp)    0.00
          ---------------------------------------------------------------------

       The step convergence is overachieved with 
       reasonable convergence on the gradient
       Convergence will therefore be signaled now


                    ***********************HURRAY********************
                    ***        THE OPTIMIZATION HAS CONVERGED     ***
                    *************************************************


    ---------------------------------------------------------------------------
                         Redundant Internal Coordinates

                          --- Optimized Parameters ---  
                            (Angstroem and degrees)

        Definition                    OldVal   dE/dq     Step     FinalVal
    ----------------------------------------------------------------------------
     1. B(F   1,H   0)                0.9238  0.000225 -0.0002    0.9236   
    ----------------------------------------------------------------------------

Geometry step timings:
Preparation and reading OPT file:      0.000 s ( 0.805 %)
Internal coordinates            :      0.000 s ( 0.218 %)
B/P matrices and projection     :      0.000 s ( 2.111 %)
Hessian update/contruction      :      0.000 s ( 3.754 %)
Making the step                 :      0.000 s ( 0.392 %)
Converting the step to Cartesian:      0.000 s ( 0.163 %)
Storing new data                :      0.000 s ( 0.196 %)
Checking convergence            :      0.000 s ( 0.087 %)
Final printing                  :      0.008 s (92.231 %)
Total time                      :      0.009 s
                 *******************************************************
                 *** FINAL ENERGY EVALUATION AT THE STATIONARY POINT ***
                 ***               (AFTER    3 CYCLES)               ***
                 *******************************************************
---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  H      0.000000    0.000000   -0.011815
  F      0.000000    0.000000    0.911815

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 H     1.0000    0     1.008    0.000000    0.000000   -0.022327
   1 F     9.0000    0    18.998    0.000000    0.000000    1.723081

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     0.923630386320     0.00000000     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 H      0   0   0     0.000000000000     0.00000000     0.00000000
 F      1   0   0     1.745408479113     0.00000000     0.00000000

Calling ../../uma.sh HF_orca_ext_EXT.extinp.tmp   ... 

Init time:    2.344 seconds
Calc time:    0.000 seconds
Import time:  1.780 seconds
Total time:   4.210 seconds

External program completed successfully in 4.865 sec

-------------------------------------------------   --------------------
FINAL SINGLE POINT ENERGY (From external program)      -100.462069891000
-------------------------------------------------   --------------------


------------------
CARTESIAN GRADIENT
------------------

   1   H   :   -0.000000000   -0.000000000    0.000001947
   2   F   :    0.000000000    0.000000000   -0.000001947

Difference to translation invariance:
           :    0.0000000000    0.0000000000    0.0000000000

Difference to rotation invariance:
           :   -0.0000000000    0.0000000000    0.0000000000

Norm of the Cartesian gradient     ...    0.0000027537
RMS gradient                       ...    0.0000011242
MAX gradient                       ...    0.0000019472
                                *** OPTIMIZATION RUN DONE ***


--------------------------------
SUGGESTED CITATIONS FOR THIS RUN
--------------------------------

Below you find a list of papers that are relevant to this ORCA run
We neither can nor want to force you to cite these papers, but we appreciate if you do
You receive ORCA, which is the product of decades of hard work by many enthusiastic individuals, for free
The only thing we kindly ask in return is that you cite our papers,
We deeply appreciate it, if you show your appreciation for ORCA by not just citing the generic ORCA reference.

Please note that relegating all ORCA citations to the supporting information does *not* help us.
SI sections are not indexed - citations you put there will not count into any citation statistics
But we need these citations in order to attract the funding resources that allow us to do what we are doing

Therefore, if you are a happy ORCA user, please consider citing a few of the papers listed below in the main body of your paper

In addition to the list printed below, the program has created the file HF_orca_ext.bibtex that contains the list in bibtex format
You can import this file easily into all common literature databanks and citation aid programs


List of essential papers. We consider these as the minimum necessary citations

  1. Neese,F.
     Software update: the ORCA program system, version 5.0
     WIRES Comput. Molec. Sci., 2022 12(1)e1606
     doi.org/10.1002/wcms.1606

List of papers to cite with high priority. The work reported in these papers was absolutely
necessary for this run to complete.
Our perspective: the developers of density functionals and basis sets usually get cited in chemistry papers
Good! But without the algorithms to do something with them, the functionals or basis sets would not do anything.
Hence, in our opinion, the algorithm design and method developments papers are equally worthy of getting cited

  1. Neese,F.
     The SHARK Integral Generation and Digestion System
     J. Comp. Chem., 2022  1-16
     doi.org/10.1002/jcc.26942

List of suggested additional citations. These are papers that are important in the 'surrounding' of 
of this run, or papers that preceded the highly important papers. If you like your results we are grateful for a citation.

  1. Neese,F.
     The ORCA program system
     WIRES Comput. Molec. Sci., 2012 2(1)73-78
     doi.org/10.1002/wcms.81
  2. Neese,F.
     Software update: the ORCA program system, version 4.0
     WIRES Comput. Molec. Sci., 2018 8(1)1-6
     doi.org/10.1002/wcms.1327
  3. Neese,F.; Wennmohs,F.; Becker,U.; Riplinger,C.
     The ORCA quantum chemistry program package
     J. Chem. Phys., 2020 152 Art. No. L224108
     doi.org/10.1063/5.0004608

List of optional additional citations

  1. Neese,F.
     Approximate second-order SCF convergence for spin unrestricted wavefunctions
     Chem. Phys. Lett., 2000 325(1-3)93-98
     doi.org/10.1016/s0009-2614(00)00662-x

Timings for individual modules:

Sum of individual times          ...       19.503 sec (=   0.325 min)
Geometry relaxation              ...        0.016 sec (=   0.000 min)   0.1 %
External energy and gradient     ...       19.487 sec (=   0.325 min)  99.9 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 19 seconds 587 msec
